dhall-kubernetes-generator/dhall-kubernetes-generator.cabal

```diff
@@ -20,14 +20,15 @@ executable dhall-kubernetes-generator
   default-extensions: DeriveDataTypeable DeriveGeneric DerivingStrategies DuplicateRecordFields GeneralizedNewtypeDeriving LambdaCase RecordWildCards ScopedTypeVariables OverloadedStrings FlexibleInstances ConstraintKinds ApplicativeDo TupleSections
   ghc-options: -Wall
   build-depends:
-    base          >= 4.8.2.0   && < 5    ,
-    aeson         >= 1.0.0.0   && < 1.5  ,
-    containers    >= 0.5.0.0   && < 0.7  ,
-    dhall         >= 1.22.0    && < 1.25 ,
-    prettyprinter >= 1.2.0.1   && < 1.3  ,
-    text          >= 0.11.1.0  && < 1.3  ,
-    turtle        >= 1.5.0     && < 1.6  ,
-    vector        >= 0.11.0.0  && < 0.13
+    base                 >= 4.8.2.0   && < 5    ,
```

Comment:
we can remove this diff i guess

Reply:
I did have to add optparse-applicative as a dependency, then there was a whitespace change to make it line up all pretty.  Is that what you are referring to?

Reply:
oh didn't notice that, nevermind